From 355e96d89fb8ec649f4fce7650cbdd5a684035fb Mon Sep 17 00:00:00 2001
From: Patrick J. Volkerding <volkerdi@slackware.com>
Date: Thu May 23 18:30:51 UTC 2024
Subject: [PATCH] Ignore .log extension

Without this patch, a log file (which should be a text file) that is
compressed will not be uncompressed and you'll get the usual warning
from less that the content might be binary, and the log file will not
be visible. Removing the special handling for .log, everything works.

--- ./lesspipe.sh.orig	2024-05-10 13:48:19.000000000 -0500
+++ ./lesspipe.sh	2024-05-23 13:28:25.917030712 -0500
@@ -140,7 +140,7 @@
 		if [[ $fchar != binary ]]; then
 		# text only file formats
 			case "$fext" in
-				html|htm|xml|pod|log)
+				html|htm|xml|pod)
 					ftype="$fext" ;;
 				pm)
 					ftype=perl ;;
